services/core-api/app/api/mines/project_summary/models/project_summary.py

```diff
@@ -0,0 +1,150 @@
+from sqlalchemy.ext.hybrid import hybrid_property
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.ext.associationproxy import association_proxy
+
+from sqlalchemy.schema import FetchedValue
+from app.extensions import db
+
+from app.api.utils.models_mixins import SoftDeleteMixin, AuditMixin, Base
+from app.api.mines.project_summary.models.project_summary_document_xref import ProjectSummaryDocumentXref
+from app.api.mines.documents.models.mine_document import MineDocument
+from app.api.parties.party.models.party import Party
+
+
+class ProjectSummary(SoftDeleteMixin, AuditMixin, Base):
+    __tablename__ = 'project_summary'
+
+    project_summary_guid = db.Column(
+        UUID(as_uuid=True), primary_key=True, server_default=FetchedValue())
+    project_summary_id = db.Column(
+        db.Integer, server_default=FetchedValue(), nullable=False, unique=True)
+    project_summary_description = db.Column(db.String, nullable=True)
+    project_summary_date = db.Column(db.DateTime, nullable=True)
+    project_summary_lead_party_guid = db.Column(
+        UUID(as_uuid=True), db.ForeignKey('party.party_guid'))
+
+    mine_guid = db.Column(UUID(as_uuid=True), db.ForeignKey('mine.mine_guid'), nullable=False)
+    status_code = db.Column(
+        db.String,
+        db.ForeignKey('project_summary_status_code.project_summary_status_code'),
+        nullable=False)
+
+    project_summary_lead = db.relationship(
+        'Party',
+        lazy='select',
+        primaryjoin='Party.party_guid == ProjectSummary.project_summary_lead_party_guid')
+
+    # Note there is a dependency on deleted_ind in mine_documents
+    documents = db.relationship('ProjectSummaryDocumentXref', lazy='select')
+    mine_documents = db.relationship(
+        'MineDocument',
+        lazy='select',
+        secondary='project_summary_document_xref',
+        secondaryjoin=
+        'and_(foreign(ProjectSummaryDocumentXref.mine_document_guid) == remote(MineDocument.mine_document_guid), MineDocument.deleted_ind == False)'
+    )
+
+    mine_table = db.relationship('Mine', lazy='joined')
```

Comment:
I think I would just call this mine instead of mine_table

Reply:
Unused references, so I just removed it.